### DISCUSSION TITLE
Copilot Refund

### DISCUSSION THREAD

**Discussion by b1uekuns** [community/community#113588] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Because I didn't know how to use the student account before, I just sent my student card and thought that by creating a business, I would be able to use the copilot for free. So I didn't pay attention to the copilot deadline and suddenly $19 was deducted from my account the other day. I was confused and immediately sent a ticket to refund the money, but it's been 3 days and I haven't received a response. 
Can you help me cancel the copilot and refund the money? For student like me, that is really a huge amount of money. Look forward to the help. 
Thank you
My ticket #2681816

**Comment by b1uekuns** [comment]
@Akash1134 help me

**Comment by davecheney** [comment, marked as answer]
@b1uekuns please contact support, we cannot assist you with billing issues via this forum. Thank you.
